README.md

```diff
@@ -43,7 +43,7 @@
 	}
 
 
-Fragment继承BaseFragment
+Fragment继承RootFragment
```

Comment:
I think highlighted[`RootFragment`](https://github.com/Mr-wangyong/FragmentStack/blob/master/stacklibrary/src/main/java/com/mrwang/stacklibrary/RootFragment.java) would be better.